tools/sil-opt/SILOpt.cpp

```diff
@@ -306,7 +310,7 @@ int main(int argc, char **argv) {
   Invocation.getLangOptions().DisableAvailabilityChecking = true;
   Invocation.getLangOptions().EnableAccessControl = false;
   Invocation.getLangOptions().EnableObjCAttrRequiresFoundation = false;
-  Invocation.getLangOptions().EnableObjCInterop =
+  Invocation.getLangOptions().EnableObjCInterop = EnableObjCInterop ? true :
```

Comment:
How hard would it be to make this entirely `EnableObjCInterop`?  It would be nice to have the ability to disable ObjC interop on Darwin as well (to test).

Reply:
I'm not an expert at LLVM command-line options parsing. If this is blocking review approval then yes, I can look into that.

Reply:
A quick look would be nice, if its difficult, then a SR would be sufficient.